### PR TITLE
Update for the new command line tool

### PR DESCRIPTION
**Pull request by alparslanavci**
This PR contains updates for the new cli implementation.

### CODE REVIEW

File: hazelcast.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Hazelcast < Formula
+  desc "Tool to run Hazelcast IMDG member instances locally"
+  homepage "https://github.com/hazelcast/hazelcast-command-line"
+  url "https://github.com/hazelcast/hazelcast-command-line/releases/download/v0.2-beta/hazelcast-command-line-0.2-beta.tar.gz"
+  sha256 "04bd3768769e81d60adaf456361690668ed3e71b221a0224bb398ed9bd2eb93f"
+
+  bottle :unneeded
+
+  depends_on "coreutils"
+  depends_on :java => "1.7+"
```
Comment (googlielmo) [inline hazelcast.rb]: probably better to bump to "1.8+" now that we're at it

File: hazelcast.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  def install
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/hazelcast"]
+    prefix.install_metafiles
+  end
+
```
Comment (googlielmo) [inline hazelcast.rb:17]: Could you please add and adapt the `test` from the old version? it'll be needed if / when we ask the main brew repo to host our formula. It basically starts an instance, checks status, and finally shuts it down.

**Review by googlielmo — CHANGES_REQUESTED** [review]
A (hopefully) quick CR. Other than that 👍 Thanks!

File: hazelcast.rb (hunk 1 above)
Comment (alparslanavci) [reply hazelcast.rb]: done 👍

File: hazelcast.rb (hunk 2 above)
Comment (alparslanavci) [reply hazelcast.rb:17]: The issue on tests is that now we are not printing anything specific to assert for. But I will create an GH issue for the tests, and work on them later.

Comment (googlielmo) [reply hazelcast.rb:17]: Ah I see! Fine enough then, thanks!!

**Review by googlielmo — APPROVED** [review]
👍 LGTM

**Pull request merged**